Source/UI/CommonUI/UIStyle/CKOUIStyle+UIFont.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+/// Default Checkout UI Style
+public enum CKOUIStyle {}
```

Comment:
We will probably have to change this CKO. I know you've already opened the followup PR, but maybe add a task so on completion of all UI work we rename this to something else?


Unless if @deepesh-vasthimal-cko is happy for us to use CKO?